status/statusgo_backend_new/accounts.nim

```diff
@@ -0,0 +1,7 @@
+import json
+import ./core, ./response_type
+
+export response_type
+
+proc getAccounts*(): RpcResponse[JsonNode] {.raises: [Exception].} =
```

Comment:
Look at the prefix of statusgo, it is located in accounts, not wallet. This has nothing to do with wallet per se.

The wallet will use the accounts, but this accounts and wallet are different

Reply:
Yse, I see that prefix in `status-go`, but it's not a must that we need to follow `status-go` and also I think this is a wrong named method in `status-go` since it doesn't do what its name says, a correct name should be `walletAccounts`, by my opinion but also we include more people @richard-ramos to give their opinion.

The thing is that `accounts_getAccounts` call returns an array of accounts displayed in the wallet, see properties of each sample in that array like wallet `type`, may be `watch`, `generated`... also there is a `name` which refers to the name of an account in the wallet and so on... so these are not user account related properties, but wallet account related.

Another thing if method is called `accounts_getAccounts` it should return list of user accounts, but why would we need it when only one account may be logged in at the same moment. So by my opinion this is not a correctly named api point.